client/components/advanced-credentials/verification/index.tsx

```diff
@@ -18,8 +18,6 @@ interface Props {
 	onReview: () => void;
 }
 
-const ERROR_CODES_FOR_BLOCKED_REQUEST = [ 'service_unavailable', 'invalid_credentials' ];
-
```

Comment:
Is this change relevant to this PR? It removes special error reporting for these two errors.